pyEpiabm/pyEpiabm/sweep/spatial_sweep.py

```diff
@@ -118,13 +123,31 @@ def __call__(self, time: float):
                     max_weight = np.nanmax(distance_weights)
                     distance_weights = np.nan_to_num(distance_weights,
                                                      nan=max_weight)
-                cell_list = random.choices(possible_infectee_cells,
-                                           weights=distance_weights,
-                                           k=number_to_infect)
+                # Use of the cutoff distance idea from CovidSim.
+                cutoff = Parameters.instance().infection_radius
+                distance_weights = [weight if (cutoff > 1/weight) else 0 for
+                                    weight in distance_weights]
```

Comment:
```suggestion
                distance_weights = [weight if (cutoff > 1/weight) else 0
                                    for weight in distance_weights]
```
I believe this is slightly better practice for the line split?